prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,14 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log(languages);
+
+const firstElement = languages[0];
+console.log(firstElement);
+
+const thirdElement = languages[2];
+console.log(thirdElement);
+
+const length = languages.length;
+console.log(length);
+
+const lastElement = languages[5];
```

Comment:
you should be using the `length` variable here to target the last index of the array within the brackets. Let me know if you have any questions!

Reply:
My bad I thought on step 6 i was suppose to "declare a variable named length and assign it the length of the array using the appropriate array property."?

Reply:
Sorry i dont understand what I did wrong, I thought i was following instructions